docker-compose.yml

```diff
@@ -152,6 +153,30 @@ services:
       args:
         BASE_IMAGE: 'sentry-onpremise-local'
     command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
+  reverse_proxy:
+    ports:
+      - '9000:80/tcp'
+    image: "nginx:1.16.1"
+    volumes:
+      - type: bind
+        source: ./reverse_proxy_config/nginx.conf
+        target: /etc/nginx/nginx.conf
+    depends_on:
+      - web
+      - relay
+  relay:
+    image: "us.gcr.io/sentryio/relay:latest"
+    ports:
+      - '3000:3000/tcp'
+    command: 'run --config /etc/relay'
+    volumes:
+      - type: bind
+        source: ./relay_config
+        target: /etc/relay
+    depends_on:
+      - kafka
+      - redis
+      - web
```

Comment:
Why do we depend on `web` service? I don't think we do.
```suggestion
```

Reply:
Relay requires this for authentication and to pull project configs.

Reply:
We chatted on this with @RaduW -- Unless this is a hard-requirement for the service to run, we'd rather not list it as when we need to run a relay command like `docker-compose run --rm relay credentials`, it will then try to bring up the Sentry web service, which depends on Postgres, Kafka, Snuba etc. which is something we don't really need.

Reply:
Removed web dependency as we do retries for authentication with web so it is not critical that web is already up by the time relay tries to authenticate with it.